### ISSUE TITLE
Add website accessibility statement by 23 Sept 2020

### ISSUE THREAD

**Issue by rainsworth** [softwaresaved/lowfat#585]
@selina-aragon has run a partial accessibility test of the lowfat website and has written up a statement (I will share the link to the Google doc via the #lowfat Slack channel).

- [x] We need to read through the statement and complete the missing information
  - [x] Contact details for Feedback and contact information **[Action: @rainsworth + @shoaibsufi]** 
  - [x] Contact details for Reporting accessibility problems with this website **[Action: @rainsworth + @shoaibsufi]**
  - [x] Date for when we will fix the accessibility issues of the website **[Action: @jag1g13]** 
- [x] The statement needs to be published before the 23rd September in a visible part of the website (e.g. the footer). This is an interim statement that we’ll need to update once the Disability Office at Edinburgh Uni runs its own check (probably in two years or more from now). **[Action: @jag1g13]** 
- [x] Selina is still to run a test using Little Forest and will update the statement once she has done that. She can share the accessibility report from Little Forest with us as a csv file. That’ll allow us to see each page and its issues. **[Action: @selina-aragon]**
- [ ] We should come up with a plan on how to fix the issues (if they are indeed fixable) by the date that we’ve set. This date can even be a year or two years from now, but we’ll need to make the necessary improvements as we move forward. **[Action: @jag1g13 + @rainsworth]** 
- [ ] Once we have a plan, we need to keep @selina-aragon in the loop.  **[Action: @rainsworth]**

**Comment by jag1g13** [comment]
Time has been scheduled in April 2021 to address the accessibility issues identified in the accessibility report
